crawler/init_freq_dict.py

```diff
@@ -0,0 +1,36 @@
+from data import *
```

Comment:
This file, init_freq_dict.py, seems somewhat extraneous. Project could use some additional organization, as it's not immediately obvious what every file does and why that is necessary.